nginx/Dockerfile

```diff
@@ -23,8 +23,15 @@
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
 FROM nginx:perl
+
 RUN rm /etc/nginx/conf.d/default.conf
 ADD cernopendata.conf /etc/nginx/conf.d/
 
+# Turn off caching in development deployment
+RUN if [ -z "$DEBUG" ]; then \
```

Comment:
Maybe remove the whole `sed` block, since people will use port 5000 for direct developments..